metpy/io/_surface_station_data.py

```diff
@@ -0,0 +1,160 @@
+from collections import defaultdict, namedtuple
+import csv
+import logging
+
+
+from metpy.cbook import get_test_data
+
+log = logging.getLogger("stations")
+log.addHandler(logging.StreamHandler())  # Python 2.7 needs a handler set
+log.setLevel(logging.WARNING)
+
+
+Station = namedtuple('Station', ['id', 'synop_id', 'name', 'state', 'country',
+                                 'longitude', 'latitude', 'altitude'])
+
+station_map = {}
+
+
+def to_dec_deg(dms):
+    """Converts to decimal degrees
+
+    """
+    if not dms:
+        return 0.
+    deg, minutes = dms.split()
+    side = minutes[-1]
+    minutes = minutes[:2]
+    float_deg = int(deg) + int(minutes) / 60.
+    return float_deg if side in ('N', 'E') else -float_deg
+
+
+def _read_station_table(filename=get_test_data('sfstns.tbl', as_file_obj=False)):
```

Comment:
So by having the call to `get_test_data()` here in the function `def`, this runs at import time, which essentially means that any import to metpy.io is going to need network access. That's no bueno. Instead, the standard Python idiom for that would be:
```python
def _read_station_table(filename=None):
    if filename is None:
        filename = get_test_data('sfstns.tbl', as_file_obj=False)
```